### ISSUE TITLE
Making tooltips for logos more meaningful

### ISSUE THREAD

**Issue by magdalenadrafiova** [DMPRoadmap/roadmap#2173]
From our accessibility testing it has been pointed out that we need to make the tooltips for logos more meaningful - e.g. the University of Edinburgh logo says it is a link to the University of Edinburgh home page etc.

**Comment by xsrust** [comment]
It looks like this is refering to the org-logo in the branding header:
https://github.com/DMPRoadmap/roadmap/blob/master/app/views/layouts/_branding.html.erb#L13-L17

The link is generated to the `org.target_url` field at the moment.
However, the organisations cannot currently edit this field.
@sjDCC I see two ways to proceed here,
First, we could make this editable, and also store the 'link text' as we're doing with the other links
Seccond, we could just remove this being a link, and rely on the branding links.

It looks like were going for the 2nd approach in the project at the moment, so it may be the most sane just to remove that field from the orgs table & not display any link here.

**Comment by sjDCC** [comment]
Two would be the easy option but I guess this would mean the logo wouldn't be a link back to homepage or the unis org. Actually, where does the link usually direct to? My expectation when I click the logo top left is that it will take me back to the webpage home. For TUDelft's version of DMPonline I would expect to be taken back to https://dmponline.tudelft.nl not https://www.tudelft.nl/en/

Perhaps we can find a way to specify the link in a common way e.g. "DMP home page for [INSERT UNI NAME]" rather than removing the link

**Comment by xsrust** [comment]
The link in the top-left (dmponline/roadmap logo) will still redirect to the homepage.

The link in question for this issue is that branded logo uploaded by organisations (in the example image, UoE).
Currently, that is only a link for a few historic orgs which happen to have data in that field, such as Edinburgh University.

![Screenshot 2019-09-02 at 12 44 06](https://user-images.githubusercontent.com/7352616/64112466-645f0400-cd7f-11e9-94e9-89a24c53a76c.png)


You bring up a great point on customizing the hover text for our different tenants, but that's outwith the scope of the Roadmap project.

**Comment by sjDCC** [comment]
Ah right - then I think we're safe to remove it. We don't want to encourage people getting redirected out of DMPonline / DMPTool

**Comment by johnpinto1** [comment]
Followed change suggested by @xsrust and removed the ability to click Org log link.
![My Dashboard - Chromium_152](https://user-images.githubusercontent.com/8876215/64334363-4805d600-cfd0-11e9-91f4-8eb79abb4386.png)

**Comment by sjDCC** [comment]
Excellent. Closing out

**Comment by magdalenadrafiova** [comment]
Looks great :)
